_summaries/dai2019transformer_1.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: summary
+title: Summary
+paper: dai2019transformer
+# Please fill out info below
+author: slycane9
+score: 9
+---
+
+## Core Idea
+The authors developed a new Transformer-XL model that had two features that separated itself from other "Vanilla" Transformers of the time in regards to creating Language Models.
+
+## Implementation
+
+First, Transformer-XL used Segment-Level Recurrence and State Reuse to avoid fixed length context segments.  This is done by storing the hidden state of a segment not just for the next segment, but also for future segments afterwards.  This gives the model a sense of extended "history" as segment dependencies reach farther into future segments.  Technically, this is realized with a recurrence mechanism that links two segments together and propgates their hidden states throughout the layer.
```

Comment:
While you have an overall good high-level explanation of this, I feel like to should expand to explain some more of the technicalities. I recommend you approach this either by first stating how the basic vanilla transformer is lacking from a technical perspective and then go into detail about how the authors created their architecture to respond to these problems, or give a technical description of the Transformer-XL architecture and just explain why each of the changes were made.

I recommend that first approach, as the audience should already understand how Transformers work and the paper gives a very good explanation about why the basic transformer is lacking performance.